editor/import/editor_scene_importer_gltf.cpp

```diff
@@ -1434,16 +1438,9 @@ Error EditorSceneImporterGLTF::_parse_skins(GLTFState &state) {
 			skin.bones.push_back(bone);
 		}
 
-		print_verbose("glTF: Skin has skeleton? " + itos(d.has("skeleton")));
 		if (d.has("skeleton")) {
-			int skeleton = d["skeleton"];
-			ERR_FAIL_INDEX_V(skeleton, state.nodes.size(), ERR_PARSE_ERROR);
-			print_verbose("glTF: Setting skeleton skin to" + itos(skeleton));
-			skin.skeleton = skeleton;
-			if (!state.skeleton_nodes.has(skeleton)) {
-				state.skeleton_nodes[skeleton] = Vector<int>();
-			}
-			state.skeleton_nodes[skeleton].push_back(i);
+			// Do nothing. This element is ignored.
+			// https://github.com/KhronosGroup/glTF/issues/1270
```

Comment:
Should we at least `print_verbose` something about `skeleton` being preset but ignored?

Reply:
It doesn't hurt to add a print_verbose. I'll do that.

Reply:
About canonicalize gltf node names, if godot is still unable to load a model because it contains nameless nodes in the scene, at the very least, it should prompt the user warning that the gltf is required to contain names in the nodes, instead of silently failing.

Reply:
Can you make a new bug with a representative gltf2 sample?

Reply:
> 
> 
> Can you make a new bug with a representative gltf2 sample?

I've just tried again to verify the issue and it worked. It's true that I tried with a simple cube and last time I tried with a much more complex project, so maybe the model was failing for some other reason.

For now disregard this case, I'll try again later when the current issues are fixed.